vernac/declaremods.mli

```diff
@@ -29,11 +29,9 @@ type inline =
 
 (** Kinds of modules *)
 
-type module_kind = Module | ModType | ModAny
-
 type 'modast module_interpretor =
```

Comment:
Since declaremods is now above modintern how about removing these function arguments?

Reply:
Great idea, thanks! Done.